src/__experimental__/components/textarea/textarea.spec.js

```diff
@@ -198,6 +198,16 @@ describe('componentWillUnmount', () => {
       expect(window.removeEventListener).not.toHaveBeenCalled();
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = renderTextarea({ label: 'required', required: true });
+
+    const input = wrapper.find('textarea');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.